Compiler/compiler.cpp

```diff
@@ -28,7 +28,7 @@ void __cdecl DolphinTrace(LPCTSTR format, ...)
 	char buf[1024];
 	va_list args;
 	va_start(args, format);
-	/*w*/vsprintf(buf, format, args);
+	/*w*/vsprintf_s(buf, sizeof(buf), format, args);
```

Comment:
... and then here use the strsafe func you've used in the other cases.
Using TCHAR and either the strsafe function (or the CRT tchar equivalent) would be one less issue to deal with if the VM were ever compiled for wide chars.